{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "If you are not using GitHub [click here to open the notebook using nbviewer](http://nbviewer.jupyter.org/github/biocore/emperor/blob/new-api/examples/evident.ipynb)."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This notebook replicates part of the [E-vident](https://github.com/biocore/evident) analysis platform, allowing you to explor a series of different distance metrics, and rarefaction levels by leveraging the Jupyter interface available in **Emperor**.\n",
    "\n",
    "Before you execute this example, you need to make sure you install a few additional dependencies:\n",
    "\n",
    "```\n",
    "pip install scikit-learn ipywidgets h5py biom-format\n",
    "```\n",
    "\n",
    "Once you've done this, you will need to enable the `ipywidgets` interface, to do so, you will need to run:\n",
    "\n",
    "```\n",
    "jupyter nbextension enable --py widgetsnbextension\n",
    "```\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%matplotlib inline\n",
    "from __future__ import division\n",
    "\n",
    "# biocore\n",
    "from emperor.qiime_backports.parse import parse_mapping_file\n",
    "from emperor.qiime_backports.format import format_mapping_file\n",
    "from emperor import Emperor, nbinstall\n",
    "\n",
    "nbinstall()\n",
    "\n",
    "from skbio.stats.ordination import pcoa\n",
    "from skbio.diversity import beta_diversity\n",
    "from skbio import TreeNode\n",
    "from skbio.io.util import open_file\n",
    "\n",
    "from biom import load_table\n",
    "from biom.util import biom_open\n",
    "\n",
    "import qiime_default_reference\n",
    "\n",
    "# pydata/scipy\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "\n",
    "from scipy.spatial.distance import braycurtis, canberra\n",
    "from ipywidgets import interact\n",
    "from sklearn.metrics import pairwise_distances\n",
    "from functools import partial\n",
    "\n",
    "import warnings\n",
    "\n",
    "warnings.filterwarnings(action='ignore', category=Warning)\n",
    "\n",
    "# -1 means all the processors available\n",
    "pw_dists = partial(pairwise_distances, n_jobs=-1)\n",
    "\n",
    "def load_mf(fn):\n",
    "    with open_file(fn) as f:\n",
    "        mapping_data, header, _ = parse_mapping_file(f)\n",
    "        _mapping_file = pd.DataFrame(mapping_data, columns=header)\n",
    "        _mapping_file.set_index('SampleID', inplace=True)\n",
    "    return _mapping_file\n",
    "\n",
    "def write_mf(f, _df):\n",
    "    with open(f, 'w') as fp:\n",
    "        lines = format_mapping_file(['SampleID'] + _df.columns.tolist(),\n",
    "                                    list(_df.itertuples()))\n",
    "        fp.write(lines+'\\n')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We are going to load data from [Fierer et al. 2010](http://www.pnas.org/content/107/14/6477.full) (the data was retrieved from study [232](https://qiita.ucsd.edu/study/description/232) in [Qiita](https://qiita.ucsd.edu), remember you need to be logged in to access the study).\n",
    "\n",
    "We will load this as a [QIIME](http://qiime.org) mapping file and as a [BIOM](http://biom-format.org) OTU table."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "mf = load_mf('keyboard/mapping-file.txt')\n",
    "bt = load_table('keyboard/otu-table.biom')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now we will load a reference database using [scikit-bio](http://scikit-bio.org)'s TreeNode object. The reference itself is as provided by [Greengenes](http://greengenes.secondgenome.com/downloads)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "tree = TreeNode.read(qiime_default_reference.get_reference_tree())\n",
    "\n",
    "for n in tree.traverse():\n",
    "    if n.length is None:\n",
    "        n.length = 0"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The function `evident` uses the OTU table (`bt`), the mapping file (`mf`), and the phylogenetic tree (`tree`), to construct a distance matrix and ordinate it using principal coordinates analysis.\n",
    "\n",
    "To exercise this function, we build a small ipywidgets function that will let us experiment with a variety of rarefaction levels and distance metrics."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "def evident(n, metric):\n",
    "    rarefied = bt.subsample(n)\n",
    "    data = np.array([rarefied.data(i) for i in rarefied.ids()], dtype='int64')\n",
    "    \n",
    "    if metric in ['unweighted_unifrac', 'weighted_unifrac']:\n",
    "        res = pcoa(beta_diversity(metric, data, rarefied.ids(),\n",
    "                                  otu_ids=rarefied.ids('observation'),\n",
    "                                  tree=tree, pairwise_func=pw_dists))\n",
    "    else:\n",
    "        res = pcoa(beta_diversity(metric, data, rarefied.ids(),\n",
    "                                  pairwise_func=pw_dists))\n",
    "    return Emperor(res, mf, remote=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Note** that the ipywidgets themselves, will not be visible unless you are executing this notebook i.e. by running your own Jupyter server."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "<script type=\"text/javascript\">\n",
       "\n",
       "if ($(\"#emperor-css\").length == 0){{\n",
       "    $(\"head\").append([\n",
       "\n",
       "        '<link id=\"emperor-css\" rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/css/emperor.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/jquery-ui.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/slick.grid.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/spectrum.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/chosen.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/jquery.contextMenu.min.css\">'\n",
       "    ]);\n",
       "}}\n",
       "</script>\n",
       "\n",
       "<div id='emperor-notebook-0x62259c1f' style=\"position: relative; width:100%; height:500px;\">\n",
       "  <div class='loading' style=\"position: absolute;top: 50%;left: 50%;margin-left: -229px; margin-top: -59px; z-index: 10000;height:118px;width:458px;padding:0px\"><img src='https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/img/emperor.png' alt='Emperor resources missing. Expected them to be found in https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files'></div>\n",
       "</div>\n",
       "</div>\n",
       "\n",
       "<script type=\"text/javascript\">\n",
       "requirejs.config({\n",
       "// the left side is the module name, and the right side is the path\n",
       "// relative to the baseUrl attribute, do NOT include the .js extension\n",
       "'paths': {\n",
       "  /* jQuery */\n",
       "  'jquery': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery-2.1.4.min',\n",
       "  'jqueryui': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery-ui.min',\n",
       "  'jquery_drag': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery.event.drag-2.2.min',\n",
       "\n",
       "  /* jQuery plugins */\n",
       "  'chosen': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/chosen.jquery.min',\n",
       "  'spectrum': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/spectrum.min',\n",
       "  'position': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery.ui.position.min',\n",
       "  'contextmenu': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery.contextMenu.min',\n",
       "\n",
       "  /* other libraries */\n",
       "  'underscore': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/underscore-min',\n",
       "  'chroma': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/chroma.min',\n",
       "  'filesaver': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/FileSaver.min',\n",
       "  'blob': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/Blob',\n",
       "  'canvastoblob': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/canvas-toBlob',\n",
       "  'd3': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/d3.min',\n",
       "\n",
       "  /* THREE.js and plugins */\n",
       "  'three': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.min',\n",
       "  'orbitcontrols': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.js-plugins/OrbitControls',\n",
       "  'projector': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.js-plugins/Projector',\n",
       "  'svgrenderer': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.js-plugins/SVGRenderer',\n",
       "  'canvasrenderer': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.js-plugins/CanvasRenderer',\n",
       "\n",
       "  /* SlickGrid */\n",
       "  'slickcore': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/slick.core.min',\n",
       "  'slickgrid': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/slick.grid.min',\n",
       "  'slickformatters': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/slick.editors.min',\n",
       "  'slickeditors': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/slick.formatters.min',\n",
       "\n",
       "  /* Emperor's objects */\n",
       "  'util': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/util',\n",
       "  'model': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/model',\n",
       "  'view': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/view',\n",
       "  'controller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/controller',\n",
       "  'draw': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/draw',\n",
       "  'scene3d': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/sceneplotview3d',\n",
       "  'abcviewcontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/abc-view-controller',\n",
       "  'viewcontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/view-controller',\n",
       "  'colorviewcontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/color-view-controller',\n",
       "  'visibilitycontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/visibility-controller',\n",
       "  'scaleviewcontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/scale-view-controller',\n",
       "  'shapecontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/shape-controller',\n",
       "  'axescontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/axes-controller',\n",
       "  'shape-editor': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/shape-editor',\n",
       "  'color-editor': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/color-editor',\n",
       "  'scale-editor': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/scale-editor',\n",
       "  'shapes': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/shapes'\n",
       "},\n",
       "/*\n",
       "   Libraries that are not AMD compatible need shim to declare their\n",
       "   dependencies.\n",
       " */\n",
       "'shim': {\n",
       "  'jquery_drag': {\n",
       "    'deps': ['jquery', 'jqueryui']\n",
       "  },\n",
       "  'chosen': {\n",
       "    'deps': ['jquery'],\n",
       "    'exports': 'jQuery.fn.chosen'\n",
       "  },\n",
       "  'contextmenu' : {\n",
       "    'deps': ['jquery', 'jqueryui', 'position']\n",
       "  },\n",
       "  'filesaver' : {\n",
       "    'deps': ['blob']\n",
       "  },\n",
       "  'canvastoblob' : {\n",
       "    'deps': ['blob']\n",
       "  },\n",
       "  'orbitcontrols': {\n",
       "    'deps': ['three']\n",
       "  },\n",
       "  'projector': {\n",
       "    'deps': ['three']\n",
       "  },\n",
       "  'svgrenderer': {\n",
       "    'deps': ['orbitcontrols', 'projector']\n",
       "  },\n",
       "  'canvasrenderer': {\n",
       "    'deps': ['orbitcontrols', 'projector']\n",
       "  },\n",
       "'slickcore': ['jqueryui'],\n",
       "'slickgrid': ['slickcore', 'jquery_drag', 'slickformatters',\n",
       "              'slickeditors']\n",
       "}\n",
       "});\n",
       "\n",
       "requirejs(\n",
       "[\"jquery\", \"model\", \"controller\"],\n",
       "function($, model, EmperorController) {\n",
       "  var DecompositionModel = model.DecompositionModel;\n",
       "\n",
       "  var div = $('#emperor-notebook-0x62259c1f');\n",
       "\n",
       "  var data = {\n",
       "    // coordinates information\n",
       "    'sample_ids': [\"232.M3Rkey217\", \"232.M9Thmr217\", \"232.M3Rinr217\", \"232.M2Enter217\", \"232.M3Midr217\", \"232.M3Tkey217\", \"232.M2Rsft217\", \"232.M2Okey217\", \"232.M9Akey217\", \"232.M9Hkey217\", \"232.M9Pinl217\", \"232.M9Midl217\", \"232.M9Rinl217\", \"232.M2Midr217\", \"232.M2Lsft217\", \"232.M9Mkey217\", \"232.M3Wkey217\", \"232.M9Gkey217\", \"232.M3Midl217\", \"232.M3Pinl217\", \"232.M9Vkey217\", \"232.M9Wkey217\", \"232.M2Tkey217\", \"232.M9Dkey217\", \"232.M2Hkey217\", \"232.M2Gkey217\", \"232.M2Fkey217\", \"232.M3Lkey217\", \"232.M9Pkey217\", \"232.M2Ykey217\", \"232.M2Pinr217\", \"232.M2Ikey217\", \"232.M9Midr217\", \"232.M3Space217\", \"232.M9Rinr217\", \"232.M3Mkey217\", \"232.M3Rsft217\", \"232.M2Lkey217\", \"232.M2Space217\", \"232.M2Vkey217\", \"232.M3Gkey217\", \"232.M9Indr217\", \"232.M3Xkey217\", \"232.M3Ykey217\", \"232.M2Bkey217\", \"232.M3Pinr217\", \"232.M3Bkey217\", \"232.M2Akey217\", \"232.M3Indr217\", \"232.M3Thmr217\", \"232.M2Skey217\", \"232.M2Ckey217\", \"232.M3Ekey217\", \"232.M2Thml217\", \"232.M2Rinl217\", \"232.M3Rinl217\", \"232.M2Midl217\"],\n",
       "    'coordinates': [[-0.39731275465584676, -0.3398747714442316, 0.21423813276420228, 0.14368894936834725, -0.2175267013838561], [-0.3747380254025983, 0.34383276098710425, 0.44995680835612556, -0.12967676775482276, -0.03094418473708699], [-0.2956183553710746, -0.4617751972749308, 0.2306573501229277, 0.09931404307595774, 0.03412343292451921], [0.7414661502920146, -0.05066178699017703, 0.0787007809390951, -0.19547880565525697, 0.08644496789632417], [-0.315316554709085, -0.45453527223707624, 0.09781582591350431, -0.012695629661470694, -0.0767175202892977], [-0.4721158916555799, -0.04440128797995247, 0.0021230150484679217, 0.1604440162903506, -0.2139718401991827], [0.7011911596289349, 0.07808727951979146, -0.05610128973154075, 0.04054032699161135, 0.10456595438062863], [0.4206703513205808, -0.09797875434384719, 0.12662778120236523, -0.25181460060420185, 0.15168814508583478], [-0.3720032362373524, 0.36065040252109865, -0.7013358485847009, -0.38915495360821817, -0.02794885458661726], [-0.4916029300494533, 0.5557011993321237, -0.17197712726036102, -0.20644719784398652, -0.023461334942119806], [-0.3544443881306613, 0.22865323990043634, 0.2886102495687219, -0.03927145861929738, 0.341781156805683], [-0.4713510055924927, 0.49042334790885467, 0.2377082855448614, 0.11173044015955924, -0.1418971712971221], [-0.41314796782125396, 0.1361272637459856, 0.3868208572898354, -0.10507502320972816, -0.0891818643607367], [0.4250869808162418, -0.05519883674144645, 0.23821315151563047, 0.03151186844210059, -0.04872425490572536], [0.6297413738502248, 0.0015323635910460603, 0.047789058521073804, -0.10548702444650308, 0.11107520698517213], [-0.5307510995716759, 0.191885734342162, 0.0697816785179921, 0.03211693587174115, -0.03380382730673254], [-0.46288153938647775, -0.22628152374359636, 0.023933438061845547, 0.273066186835009, -0.5773607653169415], [-0.4543398220549696, 0.4488496983627146, -0.29254782153490294, -0.22243301788438558, 0.27781224345832467], [-0.39370255137501203, -0.05967919060083981, -0.023834075316187538, 0.19609197997409933, 0.26723568367348305], [-0.15024235469586356, -0.5431592080799796, 0.023172909923179896, 0.13433161452690057, 0.12151642685054269], [-0.4590432866473205, 0.3789351845029376, -0.012111150964893881, -0.18687317261604863, 0.28319273633157044], [-0.5151201090321709, 0.5484505680590267, -0.09852655699229444, 0.06475263474499338, -0.05485116082698302], [0.7367794326025182, 0.023698504986522217, -0.028148965542806933, -0.07332069770060795, -0.08324255426052647], [-0.5278967092670049, 0.4967100872729833, -0.15205273428789404, -0.2114890671706699, 0.14244981995669204], [0.554290013350367, -0.08377884430478298, 0.24198014667380402, -0.3256121015270384, -0.1491573433569465], [0.8140254013000198, 0.061507941440021634, 0.19892082365323424, 0.030293737047269516, -0.0926859851556095], [0.8549686752677274, -0.08778129148576046, 0.1302265399559058, -0.1601964484096587, -0.1298404571446322], [-0.33920131121548663, -0.4621545552740063, -0.12505959183088622, -0.11549250769537182, -0.19363632422737176], [-0.5643275587462971, 0.24539861662611268, -0.780465351844103, -0.4913712074007807, -0.33260759896441633], [0.5508633003449243, -0.16249216188298038, 0.09157535422164717, -0.315986981699915, -0.31667972028773395], [0.4185007732799514, -0.006095442523103826, 0.05086171345098138, -0.17190888960772316, 0.04588388157533206], [0.559592516159657, -0.22600861617589016, -0.0437987850702453, -0.15051726676369148, 0.10952839416862109], [-0.4611112743209957, 0.5961231331980115, 0.3060519386013752, 0.12905838418306786, -0.1863911557953913], [-0.538521206551702, -0.41192479505291196, -0.22603478248436365, -0.2169462603122593, 0.19615757006978574], [-0.4209450777861946, 0.5670260582674437, 0.4722588136441862, 0.08317153409060124, 0.03203287669832649], [-0.40262506753642696, -0.4493634391479594, -0.36047947208843, -0.08867675321854039, -0.0812748060790617], [-0.37380129137379225, -0.2512242371520376, -0.031710983061356834, 0.13808222495850747, 0.30754359560789596], [0.470349118543312, -0.07273130863064528, -0.05405084669231942, -0.09626782457947064, -0.05955528116323431], [0.8152415392158469, -0.10307632625475674, 0.12946717829827695, -0.3319098264639755, 0.014013783468028713], [0.8519360472477319, 0.03418565070711616, 0.04607473399906953, -0.12813589652423213, -0.011559677692293036], [-0.3748000368330374, -0.26944818728169756, 0.09649462811251425, 0.09898821918546498, -0.2289112473183779], [-0.42889297967124324, 0.5900650412849465, 0.4550988323010131, 0.11132822765215745, 0.03960379135588289], [-0.3496722995276321, -0.4115301206664267, 0.19141296939182964, 0.0639792936476054, -0.05999434051749544], [-0.4421931304690268, -0.2797744057074231, 0.014330556970692469, -0.05060492506518057, 0.32517263122846257], [0.7858565405672359, 0.05190293805267186, 0.12138509257278708, -0.02713437572450107, 0.055666967416525034], [-0.4253326140889076, -0.5317682489300446, -0.044695757719828276, 0.011178173154857928, 0.2139966322343692], [-0.4578034402900634, -0.1632166773040765, -0.048953409482334176, 0.27695248831758273, -0.1677640863885938], [0.8199708696005843, 0.07106492861509434, -0.0016327239749161003, 0.026814177593061893, -0.0033037650586669573], [-0.34769307128706584, -0.4691940238790513, -0.09410734216272608, -0.005702785496246435, 0.19933522789708458], [-0.3703095944963633, -0.37425135634220685, 0.01678470943046472, 0.2042742309734458, 0.44249410622809454], [0.6551609825748471, 0.012266821614362544, -0.20507687028721724, 0.09292969510233687, 0.009187470089479042], [1.0, 0.14845160582066486, -0.16308657967821236, 0.10237360675812683, -0.028649275686733196], [-0.5153403628498952, -0.22241440562743564, -0.21858975788166385, 0.12514079312777465, -0.28418105127683946], [0.7372500584829353, 0.22076363469782873, -0.2636527627051007, 0.47292374833890616, -0.020390441781580875], [0.5901662721539791, 0.3065235223761333, -0.3964541068325248, 0.6216178752682442, 0.07479242912946396], [-0.4879605308121766, -0.13093079592055168, -0.029289803983115835, 0.19054342094216448, -0.09635601095239446], [0.6190518729125692, 0.3138875412466413, -0.45529885657267644, 0.7384426406419299, 0.07527547174416989]],\n",
       "    'axes_names': [\"PC1\", \"PC2\", \"PC3\", \"PC4\", \"PC5\"],\n",
       "    'percents_explained': [23.32614784209994, 8.123252617227166, 4.749288624598414, 3.9044008301023982, 2.734556379636395],\n",
       "\n",
       "    // sample information\n",
       "    'metadata_headers': [\"SampleID\", \"BarcodeSequence\", \"LinkerPrimerSequence\", \"center_name\", \"center_project_name\", \"emp_status\", \"experiment_design_description\", \"key_seq\", \"library_construction_protocol\", \"linker\", \"platform\", \"region\", \"run_center\", \"run_date\", \"run_prefix\", \"samp_size\", \"sample_center\", \"sequencing_meth\", \"study_center\", \"target_gene\", \"target_subfragment\", \"age\", \"age_unit\", \"altitude\", \"anonymized_name\", \"assigned_from_geo\", \"body_habitat\", \"body_product\", \"body_site\", \"collection_timestamp\", \"country\", \"depth\", \"dna_extracted\", \"elevation\", \"env_biome\", \"env_feature\", \"env_matter\", \"has_physical_specimen\", \"host_subject_id\", \"host_taxid\", \"latitude\", \"longitude\", \"physical_specimen_remaining\", \"project_name\", \"required_sample_info_status\", \"sample_type\", \"sex\", \"taxon_id\", \"title\", \"Description\"],\n",
       "    'metadata': [[\"232.M3Rkey217\", \"AGTCCATAGCTG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Rkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Rkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Rkey\"], [\"232.M9Thmr217\", \"ACTACGTGTGGT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Thmr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Rinr217\", \"AGGTGTGATCGC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Rinr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Enter217\", \"ACGGTGAGTGTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Enter217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ente\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ente\"], [\"232.M3Midr217\", \"AGGCTACACGAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Midr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Tkey217\", \"AGTCTACTCTGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Tkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Tkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Tkey\"], [\"232.M2Rsft217\", \"ACGCTCATGGAT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Rsft217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Right_shift\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Right_shift\"], [\"232.M2Okey217\", \"ACAGTTGCGCGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Okey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Okey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Okey\"], [\"232.M9Akey217\", \"AGACCGTCAGAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Akey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Akey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Akey\"], [\"232.M9Hkey217\", \"AGAGTAGCTAAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Hkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Hkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Hkey\"], [\"232.M9Pinl217\", \"ACTACAGCCTAT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Pinl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M9Midl217\", \"ACGTGCCGTAGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Midl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M9Rinl217\", \"ACGTTAGCACAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Rinl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Midr217\", \"ACACACTATGGC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Midr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Lsft217\", \"ACGGATCGTCAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Lsft217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Left_shift\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Left_shift\"], [\"232.M9Mkey217\", \"AGCATATGAGAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Mkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Mkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Mkey\"], [\"232.M3Wkey217\", \"AGTAGTATCCTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Wkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Wkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Wkey\"], [\"232.M9Gkey217\", \"AGAGCAAGAGCA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Gkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Gkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Gkey\"], [\"232.M3Midl217\", \"AGCTCCATACAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Midl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Pinl217\", \"AGCTGACTAGTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Pinl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M9Vkey217\", \"AGCACGAGCCTA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Vkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Vkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Vkey\"], [\"232.M9Wkey217\", \"ACTCGCACAGGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Wkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Wkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Wkey\"], [\"232.M2Tkey217\", \"ACAGAGTCGGCT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Tkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Tkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Tkey\"], [\"232.M9Dkey217\", \"AGACTGCGTACT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Dkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Dkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Dkey\"], [\"232.M2Hkey217\", \"ACCAGCGACTAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Hkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Hkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Hkey\"], [\"232.M2Gkey217\", \"ACCAGACGATGC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Gkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Gkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Gkey\"], [\"232.M2Fkey217\", \"ACCACATACATC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Fkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Fkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Fkey\"], [\"232.M3Lkey217\", \"ATACTATTGCGC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Lkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Lkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Lkey\"], [\"232.M9Pkey217\", \"AGAACACGTCTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Pkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Pkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Pkey\"], [\"232.M2Ykey217\", \"ACAGCAGTGGTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Ykey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ykey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ykey\"], [\"232.M2Pinr217\", \"ACACGAGCCACA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Pinr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Ikey217\", \"ACAGTGCTTCAT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Ikey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ikey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ikey\"], [\"232.M9Midr217\", \"ACTATTGTCACG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Midr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Space217\", \"ATCGATCTGTGG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Space217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Space_bar\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Space_bar\"], [\"232.M9Rinr217\", \"ACTCACGGTATG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Rinr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Mkey217\", \"ATCACTAGTCAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Mkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Mkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Mkey\"], [\"232.M3Rsft217\", \"ATCCGATCACAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Rsft217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Right_shift\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Right_shift\"], [\"232.M2Lkey217\", \"ACCTGTCTCTCT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Lkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Lkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Lkey\"], [\"232.M2Space217\", \"ACGTACTCAGTG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Space217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Space_bar\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Space_bar\"], [\"232.M2Vkey217\", \"ACGCAACTGCTA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Vkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Vkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Vkey\"], [\"232.M3Gkey217\", \"ATAATCTCGTCG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Gkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Gkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Gkey\"], [\"232.M9Indr217\", \"ACTAGCTCCATA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Indr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Xkey217\", \"ATAGCTCCATAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Xkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Xkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Xkey\"], [\"232.M3Ykey217\", \"AGTCTCGCATAT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Ykey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ykey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ykey\"], [\"232.M2Bkey217\", \"ACGCGATACTGG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Bkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Bkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Bkey\"], [\"232.M3Pinr217\", \"AGTACGCTCGAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Pinr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Bkey217\", \"ATATGCCAGTGC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Bkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Bkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Bkey\"], [\"232.M2Akey217\", \"ACATGATCGTTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Akey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Akey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Akey\"], [\"232.M3Indr217\", \"AGGACGCACTGT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Indr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Thmr217\", \"AGCTTGACAGCT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Thmr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Skey217\", \"ACATGTCACGTG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Skey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Skey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Skey\"], [\"232.M2Ckey217\", \"ACGATGCGACCA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Ckey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ckey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ckey\"], [\"232.M3Ekey217\", \"AGTCACATCACT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Ekey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ekey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ekey\"], [\"232.M2Thml217\", \"AACGCACGCTAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Thml217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Rinl217\", \"AAGAGATGTCGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Rinl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Rinl217\", \"AGCTCTCAGAGG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Rinl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Midl217\", \"AACTGTGCGTAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Midl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"]]\n",
       "  }\n",
       "\n",
       "  var dm, ec;\n",
       "\n",
       "  function init() {\n",
       "    // Initialize the DecompositionModel\n",
       "    dm = new DecompositionModel('', data['sample_ids'],\n",
       "                                data['coordinates'],\n",
       "                                data['percents_explained'],\n",
       "                                data['metadata_headers'], data['metadata'],\n",
       "                                data['axes_names']);\n",
       "    // Initialize the EmperorController\n",
       "    ec = new EmperorController(dm, \"emperor-notebook-0x62259c1f\");\n",
       "  }\n",
       "\n",
       "  function animate() {\n",
       "    requestAnimationFrame(animate);\n",
       "    ec.render();\n",
       "  }\n",
       "  $(window).resize(function() {\n",
       "    ec.resize(div.innerWidth(), div.innerHeight());\n",
       "  });\n",
       "\n",
       "  $(function(){\n",
       "    init();\n",
       "    animate();\n",
       "\n",
       "  });\n",
       "\n",
       "}); // END REQUIRE.JS block\n",
       "</script>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<emperor.core.Emperor at 0x106b3c550>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "interact(evident, n=(200, 2000, 50),\n",
    "         metric=['unweighted_unifrac', 'weighted_unifrac', 'braycurtis', 'euclidean'],\n",
    "         __manual=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.1"
  },
  "widgets": {
   "state": {
    "2029c3ccdb744458a24c44b8ca692da3": {
     "views": []
    },
    "271bdf1eef06456c989c9fccccd08e7f": {
     "views": [
      {
       "cell_index": 10
      }
     ]
    },
    "4b6e35651bb843849df0b174bd3658eb": {
     "views": []
    },
    "5320eb810f45432da1ca6e117d11aedf": {
     "views": []
    },
    "a148d32c050c4c8a86fb5b36e2496078": {
     "views": []
    },
    "b95d614b6c874c7bb5e8016d4359d12d": {
     "views": []
    },
    "c7b931676f424db3bba24d4334118535": {
     "views": []
    },
    "e7def816c3e545ea91284f5f0c0a76cb": {
     "views": []
    }
   },
   "version": "1.1.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
