{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "If you are not using GitHub [click here to open the notebook using nbviewer](http://nbviewer.jupyter.org/github/biocore/emperor/blob/new-api/examples/jackknifed.ipynb). **Note, if the plot below doesn't load when you run this notebook locally, make sure you click `Restart & Clear all Output` in the `Kernel` menu, then save the notebook and finally reload your browser. Otherwise the plot might not load correctly.**"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This notebook creates a jackknifed PCoA plot based on multiple rarefactions."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%matplotlib inline\n",
    "from emperor import Emperor, nbinstall\n",
    "\n",
    "nbinstall()\n",
    "\n",
    "from skbio.stats.ordination import pcoa\n",
    "from skbio.diversity import beta_diversity\n",
    "\n",
    "from biom import load_table\n",
    "\n",
    "# pydata/scipy\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "\n",
    "def load_mf(fn, index='#SampleID'):\n",
    "    _df = pd.read_csv(fn, sep='\\t', dtype=str, keep_default_na=False, na_values=[])\n",
    "    _df.set_index(index, inplace=True)\n",
    "    return _df"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We are going to load data from [Fierer et al. 2010](http://www.pnas.org/content/107/14/6477.full) (the data was retrieved from study [232](https://qiita.ucsd.edu/study/description/232) in [Qiita](https://qiita.ucsd.edu), remember you need to be logged in to access the study)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "mf = load_mf('keyboard/mapping-file.txt')\n",
    "bt = load_table('keyboard/otu-table.biom')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Create 5 rarefactions and compute the Jaccard distance matrix for each resulting table."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "ordinations = []\n",
    "for r in range(5):\n",
    "    rarefied = bt.subsample(1000)\n",
    "    data = np.array([rarefied.data(i) for i in rarefied.ids()], dtype='int64')\n",
    "    \n",
    "    res = pcoa(beta_diversity('jaccard', data, rarefied.ids()))\n",
    "    \n",
    "    ordinations.append(res)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Jackknifed plots need a *master* set of coordinates where there rest of the matrices will be centered around."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "master = ordinations[0]\n",
    "jackknifed = ordinations[1:]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/yoshikivazquezbaeza/git_sw/emperor/emperor/util.py:743: EmperorWarning: Plots generated with `remote=True` using a development version of Emperor, may fail to load in the future (only the logo may be displayed instead of the plot). To avoid this, use a release version of Emperor.\n",
      "  EmperorWarning)\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "\n",
       "<script type=\"text/javascript\">\n",
       "\n",
       "if ($(\"#emperor-css\").length == 0){{\n",
       "    $(\"head\").append([\n",
       "\n",
       "        '<link id=\"emperor-css\" rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/css/emperor.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/jquery-ui.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/slick.grid.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/spectrum.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/chosen.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/jquery.contextMenu.min.css\">'\n",
       "    ]);\n",
       "}}\n",
       "</script>\n",
       "\n",
       "<div id='emperor-notebook-0x5e4f2fcb' style=\"position: relative; width:100%; height:500px;\">\n",
       "  <div class='loading' style=\"position: absolute;top: 50%;left: 50%;margin-left: -229px; margin-top: -59px; z-index: 10000;height:118px;width:458px;padding:0px\"><img src='https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/img/emperor.png' alt='Emperor resources missing. Expected them to be found in https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files'></div>\n",
       "</div>\n",
       "</div>\n",
       "\n",
       "<script type=\"text/javascript\">\n",
       "requirejs.config({\n",
       "// the left side is the module name, and the right side is the path\n",
       "// relative to the baseUrl attribute, do NOT include the .js extension\n",
       "'paths': {\n",
       "  /* jQuery */\n",
       "  'jquery': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery-2.1.4.min',\n",
       "  'jqueryui': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery-ui.min',\n",
       "  'jquery_drag': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery.event.drag-2.2.min',\n",
       "\n",
       "  /* jQuery plugins */\n",
       "  'chosen': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/chosen.jquery.min',\n",
       "  'spectrum': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/spectrum.min',\n",
       "  'position': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery.ui.position.min',\n",
       "  'contextmenu': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery.contextMenu.min',\n",
       "\n",
       "  /* other libraries */\n",
       "  'underscore': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/underscore-min',\n",
       "  'chroma': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/chroma.min',\n",
       "  'filesaver': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/FileSaver.min',\n",
       "  'blob': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/Blob',\n",
       "  'canvastoblob': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/canvas-toBlob',\n",
       "  'd3': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/d3.min',\n",
       "\n",
       "  /* THREE.js and plugins */\n",
       "  'three': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.min',\n",
       "  'orbitcontrols': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.js-plugins/OrbitControls',\n",
       "  'projector': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.js-plugins/Projector',\n",
       "  'svgrenderer': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.js-plugins/SVGRenderer',\n",
       "  'canvasrenderer': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.js-plugins/CanvasRenderer',\n",
       "\n",
       "  /* SlickGrid */\n",
       "  'slickcore': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/slick.core.min',\n",
       "  'slickgrid': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/slick.grid.min',\n",
       "  'slickformatters': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/slick.editors.min',\n",
       "  'slickeditors': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/slick.formatters.min',\n",
       "\n",
       "  /* Emperor's objects */\n",
       "  'util': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/util',\n",
       "  'model': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/model',\n",
       "  'view': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/view',\n",
       "  'controller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/controller',\n",
       "  'draw': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/draw',\n",
       "  'scene3d': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/sceneplotview3d',\n",
       "  'abcviewcontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/abc-view-controller',\n",
       "  'viewcontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/view-controller',\n",
       "  'colorviewcontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/color-view-controller',\n",
       "  'visibilitycontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/visibility-controller',\n",
       "  'scaleviewcontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/scale-view-controller',\n",
       "  'shapecontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/shape-controller',\n",
       "  'axescontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/axes-controller',\n",
       "  'shape-editor': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/shape-editor',\n",
       "  'color-editor': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/color-editor',\n",
       "  'scale-editor': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/scale-editor',\n",
       "  'shapes': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/shapes'\n",
       "},\n",
       "/*\n",
       "   Libraries that are not AMD compatible need shim to declare their\n",
       "   dependencies.\n",
       " */\n",
       "'shim': {\n",
       "  'jquery_drag': {\n",
       "    'deps': ['jquery', 'jqueryui']\n",
       "  },\n",
       "  'chosen': {\n",
       "    'deps': ['jquery'],\n",
       "    'exports': 'jQuery.fn.chosen'\n",
       "  },\n",
       "  'contextmenu' : {\n",
       "    'deps': ['jquery', 'jqueryui', 'position']\n",
       "  },\n",
       "  'filesaver' : {\n",
       "    'deps': ['blob']\n",
       "  },\n",
       "  'canvastoblob' : {\n",
       "    'deps': ['blob']\n",
       "  },\n",
       "  'slickcore': ['jqueryui'],\n",
       "  'slickgrid': ['slickcore', 'jquery_drag', 'slickformatters', 'slickeditors']\n",
       "}\n",
       "});\n",
       "\n",
       "requirejs(\n",
       "[\"jquery\", \"model\", \"controller\"],\n",
       "function($, model, EmperorController) {\n",
       "  var DecompositionModel = model.DecompositionModel;\n",
       "\n",
       "  var div = $('#emperor-notebook-0x5e4f2fcb');\n",
       "\n",
       "  var data = {\n",
       "    // coordinates information\n",
       "    'decompositions': [\n",
       "      {\n",
       "        'sample_ids': [\"232.M3Rkey217\", \"232.M9Thmr217\", \"232.M9Pinr217\", \"232.M3Rinr217\", \"232.M2Enter217\", \"232.M9Ekey217\", \"232.M3Midr217\", \"232.M3Tkey217\", \"232.M2Rsft217\", \"232.M2Okey217\", \"232.M3Ckey217\", \"232.M9Akey217\", \"232.M9Hkey217\", \"232.M9Pinl217\", \"232.M2Kkey217\", \"232.M9Midl217\", \"232.M9Rinl217\", \"232.M9Ckey217\", \"232.M2Midr217\", \"232.M2Lsft217\", \"232.M9Mkey217\", \"232.M9Enter217\", \"232.M9Bkey217\", \"232.M9Fkey217\", \"232.M2Mkey217\", \"232.M3Wkey217\", \"232.M9Gkey217\", \"232.M3Midl217\", \"232.M3Pinl217\", \"232.M9Vkey217\", \"232.M9Wkey217\", \"232.M2Tkey217\", \"232.M3Indl217\", \"232.M9Dkey217\", \"232.M2Indl217\", \"232.M2Hkey217\", \"232.M2Gkey217\", \"232.M2Jkey217\", \"232.M9Nkey217\", \"232.M2Ukey217\", \"232.M2Fkey217\", \"232.M3Lkey217\", \"232.M9Pkey217\", \"232.M2Ykey217\", \"232.M2Pinr217\", \"232.M2Ikey217\", \"232.M2Ekey217\", \"232.M9Midr217\", \"232.M3Space217\", \"232.M9Thml217\", \"232.M9Rinr217\", \"232.M3Mkey217\", \"232.M3Rsft217\", \"232.M2Lkey217\", \"232.M2Space217\", \"232.M2Vkey217\", \"232.M3Gkey217\", \"232.M9Indr217\", \"232.M3Xkey217\", \"232.M3Ykey217\", \"232.M3Vkey217\", \"232.M2Bkey217\", \"232.M3Pinr217\", \"232.M3Bkey217\", \"232.M2Akey217\", \"232.M3Indr217\", \"232.M3Thmr217\", \"232.M2Skey217\", \"232.M2Wkey217\", \"232.M2Rinr217\", \"232.M2Ckey217\", \"232.M3Ekey217\", \"232.M2Zkey217\", \"232.M2Thml217\", \"232.M2Rinl217\", \"232.M3Rinl217\", \"232.M2Midl217\"],\n",
       "        'coordinates': [[-0.4209925913414051, -0.5899201451759268, -0.04854458803326332, -0.13748774013290838, -0.12806284749246336], [-0.3048571864991666, 0.6535027835047095, -0.4403189003955411, 0.09350702079279792, -0.05239839656473033], [-0.3931406392053314, 0.6408749914134398, -0.3459073357783165, -0.1347418531813774, -0.24222620708123016], [-0.3148177939931439, -0.3137750852207863, -0.32380333608938094, 0.28373856510694606, -0.025251139626010433], [0.6702898458618767, 0.02759077448553156, -0.2080434525346929, -0.14717022471538868, -0.03552467071457001], [-0.43235699737341166, 0.4535704484141676, 0.3835360792599373, -0.07022052221300187, 0.6473519208464911], [-0.28651402419427274, -0.25141166369244955, -0.35805725321987697, 0.2222178401520069, 0.2833483179789491], [-0.4350100511470442, -0.35761861849157095, -0.11642776308664422, 0.10890550995901234, -0.10411138746315785], [0.6666713532764315, -0.07327292538748195, 0.16484855475224403, 0.45817234901785164, -0.09113342714888419], [0.4382933198479371, -0.0005078812497692553, -0.14420220142481094, -0.2313803936763929, -0.04940219225512876], [-0.3075314067985154, -0.5133675219858423, 0.22192433343623116, 0.015712034627038317, 0.03975130820650567], [-0.2554960141839766, 0.20964259109274921, 0.2786528952438309, 0.11308793113067192, 0.20436911001999308], [-0.46383145697076733, 0.5237179782860253, 0.3366591615040235, -0.0175806546347812, 0.4210847579046222], [-0.3516810143852833, 0.5878921224074004, -0.6145548970996455, -0.03242383020201315, -0.07659102509645979], [0.7256103859025386, 0.09696815529422143, -0.3375133306598328, -0.5352811622452999, 0.23578205182672168], [-0.41620177453099066, 0.6409708374083726, -0.476355981814306, 0.4493102707340806, -0.11548511004586264], [-0.350661459095282, 0.5936640658519192, -0.5023667269162455, 0.06842654803872182, -0.1542596316193951], [-0.46558233624675427, 0.486569054077339, 0.540110208660812, -0.10187112740614317, 0.042259504599596605], [0.4860997233594788, 0.1603891511889657, -0.3302927733054933, -0.2736577946602568, 0.2277917549507505], [0.6745640464439429, -0.0327621316018908, -0.07922295186188687, -0.398507176796543, -0.015068325283120998], [-0.45155072030858223, 0.4237659612323957, 0.30483072267808003, 0.03567568025712457, -0.010236609308550589], [-0.6436790912266046, 0.4447870903777984, 0.6415583807842684, -0.4210347353094003, -0.022031904495716896], [-0.5079240935503612, 0.36093370832303406, 0.7370031699445234, -0.2764977777764083, -0.3859917431138798], [-0.40615530363376295, 0.4163804987575953, 0.07290912458267274, -0.03624266785019274, -0.05050122461766671], [0.42685417997165603, 0.0369135018115311, -0.3218855767216332, -0.4673318056737851, 0.3080208260054065], [-0.3651612644804948, -0.48016087462411655, -0.10665282019127272, -0.16082043356111952, 0.09868816844563331], [-0.44905956242440387, 0.33803050037081867, 0.42889403194115483, -0.32628621621953735, -0.17786469623389506], [-0.4647873365050955, -0.3720891128907655, -0.08862073211244008, -0.1707034999557861, -0.23773047275936748], [-0.25878516395985374, -0.3757371137613795, -0.2934188829884553, 0.07588001557815331, 0.07028679919563603], [-0.43068044138522277, 0.5848564575064028, 0.28447387425527565, 0.07253613540368764, 0.24131480611931938], [-0.5028438888845871, 0.5642160625561197, 0.21845715552128073, -0.02285609773281353, 0.015764659597459006], [0.7567696453026342, -0.04219765107613673, 0.061776667031725116, -0.029763389767525746, 0.06617341717395506], [-0.447895556848754, -0.33812779738224047, -0.08517285991443271, 0.1259058227240486, -0.13090317640345916], [-0.46149081718261636, 0.3929595687257057, 0.6897230152781137, -0.4742552239843028, 0.24173705511408944], [0.5972792494978914, -0.004416306256985624, 0.08359465540691366, 0.1889744957084139, 0.19179442317484674], [0.6307673180197461, -0.017114612418296368, -0.2552718824492325, -0.3486747042156605, -0.08866630813728327], [0.7551852506753844, -0.009295792326019842, -0.06004235415668827, -0.189860434872809, -0.13553879635399457], [0.6854799073334394, 0.03738401274658326, -0.2752241089283089, -0.5666055759287146, 0.13080184640204154], [-0.44971169248393705, 0.17250915497858846, 0.2939940718378004, -0.5270598398766271, -0.23452676191015182], [0.51239879141798, 0.07730275193135869, -0.18681213114295775, -0.2786740616049347, 0.2202553605139474], [0.817590689489573, 0.07618307717712589, -0.024646775017981792, 0.0006395255376058798, -0.315230403429119], [-0.39456557560426775, -0.5085409149521828, -0.06117146835057006, 0.18154154761844965, 0.14236484012128445], [-0.3222479322925208, 0.13668023243044547, 0.5199659378830468, -0.08513243122128308, 0.20327936249210374], [0.6728809187957447, -0.09859107831848139, -0.17443842962550876, -0.48446748252022154, -0.09761616828796474], [0.5677346782226655, 0.1839383370651608, -0.21364165945185465, -0.14335116722169405, 0.44104396802540846], [0.5061324156307979, -0.11376235376827423, 0.05722810952410849, 0.06891426904909075, 0.1098768051022001], [0.6150199656041795, -0.15084837447060914, 0.39099063027045583, 0.3575808580634397, -0.23513997300544465], [-0.43884234586054766, 0.5864525314875888, -0.7352441849296233, 0.2029720711019638, -0.31336339604108365], [-0.45198325460325606, -0.6180046358307884, 0.13304345424792324, -0.28619334920110495, 0.09202706763739077], [-0.5506245238081824, 0.7209735551903765, -0.18085530030585903, 0.1895118703733766, -0.15278949690654506], [-0.24802755830553877, 0.6025645106849098, -0.4836639792668584, 0.08733798183409917, -0.5184001993917808], [-0.43751258617779304, -0.6128120185567574, -0.043834700675962256, 0.14120819362704692, 0.13179972494563322], [-0.4090547261374843, -0.47997073498822695, 0.001558838011928665, -0.04587854867768469, -0.059550800847231924], [0.4448142825740676, 0.04389676087954929, 0.14439111673853888, -0.07837141798239372, 0.24395287718824538], [0.8450707446056935, -0.034056436691501935, 0.12154298617843802, -0.13136533112936033, -0.3718114220349811], [0.9312461911169831, -0.02586279505526879, 0.010526552735298466, -0.14550413421364938, -0.31544662918091715], [-0.4690375419944641, -0.42143818918246, -0.3368980392780982, 0.41761507138945075, 0.01452352111496441], [-0.42784444190692844, 0.7996574596391691, -0.6112763081126403, 0.45890142689079755, 0.0024310873070291284], [-0.4285575583305855, -0.545738753376382, -0.005777124924211944, -0.13782319460570877, 0.025853362025031596], [-0.5003214984883692, -0.6631327779130867, -0.16413296147306863, -0.07207394645325128, 0.19622428766629074], [-0.4349062002296223, -0.2903895592830438, 0.4860382361106663, 0.02295250739477754, -0.2499310251423093], [0.8450551050936171, 0.013021841362610188, -0.19509191490535027, -0.16811819107498405, 0.021466409722620528], [-0.4116814927896277, -0.4595222951574306, -0.13290849389200995, 0.024752735231141194, -0.16657208088610695], [-0.3851626002642961, -0.592014706829601, -0.1867721828893308, -0.15896108402212353, 0.147760252412159], [0.7041352625525097, -0.04741388790034027, 0.08014917784965976, -0.02388529958443527, -0.1679166640936677], [-0.49696014253901266, -0.7424602166903577, -0.12356501350081653, 0.03823332166371217, 0.12735782863606354], [-0.43396308515269616, -0.7261251927865394, -0.19983546352551243, 0.14244717379635657, 0.036528280117015775], [0.616834152504592, -0.02333350915838379, 0.241584175929373, 0.6439999142745391, 0.06118886396131353], [0.6375422709608487, -0.09603145093948169, 0.18880461017949457, 0.19800156889712156, -0.13767534829328296], [0.5597388792261673, 0.18110950197312153, -0.040207182899708656, 0.1488438652093742, 0.7070141259053395], [0.5875053876642529, -0.10700315405796071, 0.42450096070239274, 0.4549337586132033, -0.4972340213404135], [-0.5505702161218219, -0.6861845688360136, 0.17689066515589696, -0.1866546812822354, -0.12167445198568676], [0.5539294775028367, -0.06558750083085033, 0.275521075027435, 0.15019209441900966, -0.3776512310134347], [0.6375971380189887, -0.001904750193173327, 0.4377013132007942, 0.6121544566718249, 0.09050658466262815], [0.404027190459259, 0.11755367311128528, 0.4886846713309888, 0.8448171425305088, 0.29427836945666463], [-0.43682775244298455, -0.4745302777413535, -0.4256955479981782, 0.009788792379852747, -0.14386927729141563], [0.2939729449559002, -0.030388336693880497, 0.4163009586531608, 0.8153788375865618, 0.029324936321002852]],\n",
       "        'axes_names': [\"PC1\", \"PC2\", \"PC3\", \"PC4\", \"PC5\"],\n",
       "        'percents_explained': [4.301387599732598, 2.705364632714371, 2.051049803015049, 1.843131900811226, 1.7500886503783661],\n",
       "        'ci': [[0.030933599642636866, 0.10872515313697861, 0.23637436882050572, 0.2682540856055782, 0.3275914620171997], [0.09551229876419531, 0.14818163646145788, 0.3702498784285486, 0.2774142486476241, 0.47517786277], [0.08177891259220565, 0.140059961082159, 0.5041153117508963, 0.3981036620111412, 0.30520821053580705], [0.03760230588223695, 0.11205583696567761, 0.060311413829644556, 0.26467793560236735, 0.6086491384373962], [0.11794709116121327, 0.09077332885674558, 0.20331230583098822, 0.4070645646339124, 0.3484926302453413], [0.062447608395457765, 0.12598195051984595, 0.4814998060982126, 0.23070924906545298, 0.19545774897649326], [0.0468404526851291, 0.11616782520392627, 0.0945040973502097, 0.4588153795838008, 0.5403433050418325], [0.11789134211176239, 0.1988819604227609, 0.10669223412759661, 0.28319275883505485, 0.47457979087605384], [0.10448759214234493, 0.07516203964212273, 0.32565241385104676, 0.30435336483735564, 0.3563027869167328], [0.0849443322985387, 0.03772425775463886, 0.17911504987088547, 0.29703631345365067, 0.23571112548444947], [0.07284074768188137, 0.14235689128819334, 0.18114080649388006, 0.3491558230767722, 0.9988121517238975], [0.05490902685592067, 0.06908059992925555, 0.38132495972634306, 0.33098509590584607, 0.9497462197434486], [0.06191089533997951, 0.05743630892875262, 0.20308986240319282, 0.4900714720970576, 0.2931216007023209], [0.10536067526985404, 0.06198481889711449, 0.35799390785924684, 0.3453679490666852, 0.3804571361648805], [0.19038980254127214, 0.0745395222834327, 0.12478950921802101, 0.39342598003673857, 0.2039874847093016], [0.1315500662708492, 0.22136392915406078, 0.537428527937961, 0.2846574070116613, 0.44189396321541896], [0.09594745919974734, 0.054978086142432936, 0.33655079155787726, 0.24601019258078827, 0.7670779539635706], [0.05199396686568125, 0.2476355310356615, 0.28548839341908805, 0.7037912847382326, 0.5058851926539137], [0.13364414085056076, 0.13287382769794093, 0.13533147963724362, 0.5377491940604697, 0.30490393116328707], [0.09994345990853959, 0.1529356842844185, 0.3013506959215211, 0.2980966761049294, 0.31840523073124916], [0.1331436638667074, 0.18008301528326698, 0.298748412849534, 0.2322678595632371, 0.5949409179320664], [0.13324063177311396, 0.1685682459529143, 0.30013370964641134, 0.3983615152145267, 0.28572078299237547], [0.0891863038502771, 0.17327435344571962, 0.15526858908910035, 0.26623796055292315, 0.25613041803773573], [0.0760537927335887, 0.14635389277120014, 0.15716872159355774, 0.16704695284228896, 0.4843364198163464], [0.11496558095294257, 0.10512644612051239, 0.30896638743509525, 0.5311714693711027, 0.2229192088590424], [0.06822488458378195, 0.19764950178114088, 0.28911656684368364, 0.21356606887056478, 0.37502145463221515], [0.11394362375465156, 0.15915807238197677, 0.3330122793808376, 0.5283671781002381, 0.6465484407814448], [0.15437729940832745, 0.250439191604382, 0.4222244864420929, 0.4492580411829413, 0.6950236165918923], [0.05505840784083876, 0.15982958429785749, 0.20988092292527527, 0.2891143342701791, 0.4021515505925913], [0.10387515214970466, 0.08054065211128647, 0.29746135185152656, 0.08132980366428466, 0.7775596788488446], [0.09761254560700999, 0.3295498501472725, 0.18697610706483073, 0.2693928953123335, 0.7686952924753538], [0.18116937654172693, 0.17917665711415226, 0.14635515645246858, 0.5376482467709095, 0.44581088049715994], [0.033263022788443064, 0.03439776423438318, 0.21362431051786435, 0.22580539671764477, 0.35424554498064525], [0.09455962123266548, 0.21568355823529495, 0.256815715420314, 0.3248757240989626, 0.4532107672559549], [0.06813513785559877, 0.052216460803936895, 0.07692494148835675, 0.22901075334028426, 0.15840027902766607], [0.07012159803784224, 0.06787208199880934, 0.2571089348143891, 0.2453498965864353, 0.3973553769335546], [0.13452710413844227, 0.06968577425360684, 0.1302203157979692, 0.17124290147294519, 0.3231013539807872], [0.08383036776830854, 0.1548742066261789, 0.1729108788023005, 0.37010203951371945, 0.16665590791411938], [0.09810021230656685, 0.15081520807142085, 0.5096668840375487, 0.4922771053760653, 0.4265925827514827], [0.07704529491002376, 0.08734977883576509, 0.23737720488686942, 0.20397656159989513, 0.29619567622936643], [0.11093466334031354, 0.05444681423521246, 0.14090722434063016, 0.4820844276472533, 0.24280148456316844], [0.08059511192960406, 0.07597301463052386, 0.31334086938897254, 0.4190422503107327, 0.6367036650998397], [0.048402782009266176, 0.1314655809194728, 0.24023038210727843, 0.25572596673450787, 0.9061341154448457], [0.07999274534790668, 0.07988799145881355, 0.21993518389875488, 0.18531139577820266, 0.3240223289524584], [0.1827480815290506, 0.03551055473622888, 0.12138809212830065, 0.7433436367838278, 0.7339912298249548], [0.11090073290839947, 0.07395110756711004, 0.09721078703537755, 0.38476727299295904, 0.4424564873997313], [0.08520830340953822, 0.06293982321484756, 0.24957136920451567, 0.3569868497480176, 0.13444358129982675], [0.0825164910706433, 0.12678077073546756, 0.43150417281424736, 0.34028561202002805, 0.5456376407410292], [0.17490637122065766, 0.1921110899928864, 0.3817718880410341, 0.3929433966723493, 0.2641010628848526], [0.12366683960623448, 0.18394391255905163, 0.265674599975398, 0.25622275148251994, 0.5651533666414872], [0.10073126937359272, 0.14873150032894822, 0.38128600422561604, 0.38088722398005664, 0.5622730575552447], [0.07639326848567479, 0.07808829029627984, 0.34429809801615263, 0.2218243759339154, 0.3405866521171403], [0.07011402727268468, 0.07298358750398148, 0.18222953221672672, 0.23499145168219088, 0.10123556679436968], [0.08571832282593639, 0.08555979128633666, 0.08172787808399146, 0.342223938952858, 0.4066672017333537], [0.14776501025995847, 0.0899013828224553, 0.20557509073367344, 0.36514614151472635, 0.2536386094188481], [0.17188452220754202, 0.047848624929657683, 0.19254033692392813, 0.22874896851961993, 0.4089341530826596], [0.07956834929550616, 0.06841576313543263, 0.168307057604843, 0.3114311950975316, 0.25969255206577213], [0.18131930439155614, 0.12248328095813366, 0.2616220885923267, 0.18745778456207407, 0.6426986567867254], [0.05705622396502308, 0.06487989737784239, 0.1274103292101822, 0.3131430204046524, 0.46655142155011026], [0.09975073955014835, 0.1779415118098182, 0.33887122663496116, 0.43876099904309407, 0.6263815500222382], [0.10085943506854844, 0.10891796375978921, 0.3584007701832549, 0.26294651318426254, 0.45656530103698767], [0.16170667973225106, 0.03434084588432811, 0.13559159667448725, 0.09917496558086152, 0.3744039778467781], [0.038278520010951333, 0.21879388876530903, 0.16161509332335977, 0.1622837385978353, 0.5014191651627513], [0.13036991502144735, 0.1521284106258105, 0.1822842487122499, 0.1256351204997502, 0.6235958554738734], [0.08686798742937751, 0.10052318104555037, 0.11532609876637397, 0.21568338146192456, 0.1750884572243448], [0.14957924347850615, 0.15705969846031964, 0.22123866539552883, 0.1239048385003355, 0.3175831820971991], [0.09435234310640822, 0.17204887534868518, 0.3257080450582964, 0.10371200755199339, 0.24965612974734963], [0.07037922329096824, 0.13715706807943903, 0.27952404263290587, 0.3247707250272677, 0.37877114228393327], [0.13743575535948493, 0.05730768494686683, 0.14068972297248647, 0.32699559302135855, 0.09173812657156329], [0.1138291611991375, 0.049203869782157655, 0.17518228393588497, 0.5378685548210452, 0.6012608647712365], [0.07247282936598431, 0.027557049630216213, 0.28046679133097074, 0.581936410586891, 0.29964075902052784], [0.11432051175790658, 0.1104351077152016, 0.31820132641329024, 0.3486889781983327, 0.5879560112282505], [0.05559445603585633, 0.05284063432031113, 0.1642645813625905, 0.16226060295831854, 0.2922853090633313], [0.12040934385003632, 0.08447149615492658, 0.2560409605750354, 0.22174009222049473, 0.10547243639746676], [0.08828327954046672, 0.05552744704506225, 0.45856494183192775, 0.2421320422978912, 0.5924891667870829], [0.13565403346081578, 0.19801420998725783, 0.29725469681062067, 0.193369813996355, 0.6501116312670452], [0.046328692067134025, 0.04777697442198152, 0.3422780882492199, 0.20843811365674902, 0.7703216421409438]],\n",
       "        'type': 'ordination'\n",
       "      }\n",
       "    ],\n",
       "\n",
       "    // sample information\n",
       "    'metadata_headers': [\"SampleID\", \"BarcodeSequence\", \"LinkerPrimerSequence\", \"center_name\", \"center_project_name\", \"emp_status\", \"experiment_design_description\", \"key_seq\", \"library_construction_protocol\", \"linker\", \"platform\", \"region\", \"run_center\", \"run_date\", \"run_prefix\", \"samp_size\", \"sample_center\", \"sequencing_meth\", \"study_center\", \"target_gene\", \"target_subfragment\", \"age\", \"age_unit\", \"altitude\", \"anonymized_name\", \"assigned_from_geo\", \"body_habitat\", \"body_product\", \"body_site\", \"collection_timestamp\", \"country\", \"depth\", \"dna_extracted\", \"elevation\", \"env_biome\", \"env_feature\", \"env_matter\", \"has_physical_specimen\", \"host_subject_id\", \"host_taxid\", \"latitude\", \"longitude\", \"physical_specimen_remaining\", \"project_name\", \"required_sample_info_status\", \"sample_type\", \"sex\", \"taxon_id\", \"title\", \"Description\"],\n",
       "    'metadata': [[\"232.M3Rkey217\", \"AGTCCATAGCTG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Rkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Rkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Rkey\"], [\"232.M9Thmr217\", \"ACTACGTGTGGT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Thmr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M9Pinr217\", \"ACTCAGATACTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Pinr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Rinr217\", \"AGGTGTGATCGC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Rinr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Enter217\", \"ACGGTGAGTGTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Enter217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ente\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ente\"], [\"232.M9Ekey217\", \"ACTCTTCTAGAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Ekey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ekey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ekey\"], [\"232.M3Midr217\", \"AGGCTACACGAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Midr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Tkey217\", \"AGTCTACTCTGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Tkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Tkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Tkey\"], [\"232.M2Rsft217\", \"ACGCTCATGGAT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Rsft217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Right_shift\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Right_shift\"], [\"232.M2Okey217\", \"ACAGTTGCGCGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Okey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Okey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Okey\"], [\"232.M3Ckey217\", \"ATAGGCGATCTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Ckey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ckey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ckey\"], [\"232.M9Akey217\", \"AGACCGTCAGAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Akey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Akey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Akey\"], [\"232.M9Hkey217\", \"AGAGTAGCTAAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Hkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Hkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Hkey\"], [\"232.M9Pinl217\", \"ACTACAGCCTAT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Pinl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Kkey217\", \"ACCTCGATCAGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Kkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Kkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Kkey\"], [\"232.M9Midl217\", \"ACGTGCCGTAGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Midl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M9Rinl217\", \"ACGTTAGCACAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Rinl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M9Ckey217\", \"AGCACACCTACA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Ckey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ckey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ckey\"], [\"232.M2Midr217\", \"ACACACTATGGC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Midr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Lsft217\", \"ACGGATCGTCAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Lsft217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Left_shift\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Left_shift\"], [\"232.M9Mkey217\", \"AGCATATGAGAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Mkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Mkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Mkey\"], [\"232.M9Enter217\", \"AGCGAGCTATCT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Enter217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ente\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ente\"], [\"232.M9Bkey217\", \"AGCAGCACTTGT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Bkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Bkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Bkey\"], [\"232.M9Fkey217\", \"AGAGAGCAAGTG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Fkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Fkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Fkey\"], [\"232.M2Mkey217\", \"ACGCTATCTGGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Mkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Mkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Mkey\"], [\"232.M3Wkey217\", \"AGTAGTATCCTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Wkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Wkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Wkey\"], [\"232.M9Gkey217\", \"AGAGCAAGAGCA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Gkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Gkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Gkey\"], [\"232.M3Midl217\", \"AGCTCCATACAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Midl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Pinl217\", \"AGCTGACTAGTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Pinl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M9Vkey217\", \"AGCACGAGCCTA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Vkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Vkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Vkey\"], [\"232.M9Wkey217\", \"ACTCGCACAGGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Wkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Wkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Wkey\"], [\"232.M2Tkey217\", \"ACAGAGTCGGCT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Tkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Tkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Tkey\"], [\"232.M3Indl217\", \"AGCTATCCACGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Indl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M9Dkey217\", \"AGACTGCGTACT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Dkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Dkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Dkey\"], [\"232.M2Indl217\", \"AACTCGTCGATG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Indl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Hkey217\", \"ACCAGCGACTAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Hkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Hkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Hkey\"], [\"232.M2Gkey217\", \"ACCAGACGATGC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Gkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Gkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Gkey\"], [\"232.M2Jkey217\", \"ACCGCAGAGTCA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Jkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Jkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Jkey\"], [\"232.M9Nkey217\", \"AGCAGTCGCGAT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Nkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Nkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Nkey\"], [\"232.M2Ukey217\", \"ACAGCTAGCTTG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Ukey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ukey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ukey\"], [\"232.M2Fkey217\", \"ACCACATACATC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Fkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Fkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Fkey\"], [\"232.M3Lkey217\", \"ATACTATTGCGC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Lkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Lkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Lkey\"], [\"232.M9Pkey217\", \"AGAACACGTCTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Pkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Pkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Pkey\"], [\"232.M2Ykey217\", \"ACAGCAGTGGTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Ykey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ykey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ykey\"], [\"232.M2Pinr217\", \"ACACGAGCCACA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Pinr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Ikey217\", \"ACAGTGCTTCAT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Ikey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ikey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ikey\"], [\"232.M2Ekey217\", \"ACACTGTTCATG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Ekey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ekey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ekey\"], [\"232.M9Midr217\", \"ACTATTGTCACG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Midr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Space217\", \"ATCGATCTGTGG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Space217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Space_bar\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Space_bar\"], [\"232.M9Thml217\", \"ACGTCTGTAGCA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Thml217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M9Rinr217\", \"ACTCACGGTATG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Rinr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Mkey217\", \"ATCACTAGTCAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Mkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Mkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Mkey\"], [\"232.M3Rsft217\", \"ATCCGATCACAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Rsft217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Right_shift\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Right_shift\"], [\"232.M2Lkey217\", \"ACCTGTCTCTCT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Lkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Lkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Lkey\"], [\"232.M2Space217\", \"ACGTACTCAGTG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Space217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Space_bar\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Space_bar\"], [\"232.M2Vkey217\", \"ACGCAACTGCTA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Vkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Vkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Vkey\"], [\"232.M3Gkey217\", \"ATAATCTCGTCG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Gkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Gkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Gkey\"], [\"232.M9Indr217\", \"ACTAGCTCCATA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Indr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Xkey217\", \"ATAGCTCCATAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Xkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Xkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Xkey\"], [\"232.M3Ykey217\", \"AGTCTCGCATAT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Ykey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ykey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ykey\"], [\"232.M3Vkey217\", \"ATATCGCTACTG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Vkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Vkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Vkey\"], [\"232.M2Bkey217\", \"ACGCGATACTGG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Bkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Bkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Bkey\"], [\"232.M3Pinr217\", \"AGTACGCTCGAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Pinr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Bkey217\", \"ATATGCCAGTGC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Bkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Bkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Bkey\"], [\"232.M2Akey217\", \"ACATGATCGTTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Akey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Akey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Akey\"], [\"232.M3Indr217\", \"AGGACGCACTGT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Indr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Thmr217\", \"AGCTTGACAGCT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Thmr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Skey217\", \"ACATGTCACGTG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Skey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Skey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Skey\"], [\"232.M2Wkey217\", \"ACACTAGATCCG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Wkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Wkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Wkey\"], [\"232.M2Rinr217\", \"ACACATGTCTAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Rinr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Ckey217\", \"ACGATGCGACCA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Ckey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ckey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ckey\"], [\"232.M3Ekey217\", \"AGTCACATCACT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Ekey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ekey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ekey\"], [\"232.M2Zkey217\", \"ACGACGTCTTAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Zkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Zkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Zkey\"], [\"232.M2Thml217\", \"AACGCACGCTAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Thml217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Rinl217\", \"AAGAGATGTCGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Rinl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Rinl217\", \"AGCTCTCAGAGG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Rinl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Midl217\", \"AACTGTGCGTAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Midl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"]],\n",
       "    'settings': {}\n",
       "  }\n",
       "\n",
       "  var dm, ec;\n",
       "\n",
       "  function init() {\n",
       "    var decomp = data['decompositions'][0];\n",
       "\n",
       "    // Initialize the DecompositionModel\n",
       "    dm = new DecompositionModel(decomp, data['metadata_headers'],\n",
       "                                data['metadata']);\n",
       "    // Initialize the EmperorController\n",
       "    ec = new EmperorController(dm, \"emperor-notebook-0x5e4f2fcb\");\n",
       "  }\n",
       "\n",
       "  function animate() {\n",
       "    requestAnimationFrame(animate);\n",
       "    ec.render();\n",
       "  }\n",
       "  $(window).resize(function() {\n",
       "    ec.resize(div.innerWidth(), div.innerHeight());\n",
       "  });\n",
       "\n",
       "  $(function(){\n",
       "    init();\n",
       "    animate();\n",
       "\n",
       "    ec.ready = function () {\n",
       "      // any other code that needs to be executed when emperor is loaded should\n",
       "      // go here\n",
       "      ec.loadConfig(data['settings']);\n",
       "    }\n",
       "  });\n",
       "\n",
       "}); // END REQUIRE.JS block\n",
       "</script>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<emperor.core.Emperor at 0x1110ba0f0>"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Emperor(master, mf, jackknifed=jackknifed)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.1"
  },
  "widgets": {
   "state": {},
   "version": "1.1.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
