{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "If you are not using GitHub [click here to open the notebook using nbviewer](http://nbviewer.jupyter.org/github/biocore/emperor/blob/new-api/examples/keyboard.ipynb)."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "source": [
    "# Emperor's Python API\n",
    "\n",
    "**This notebook demonstrate Emperor's new Python API, which can and will change as we continue to exercise this interface, for more information, have a look at the [pull request here](https://github.com/biocore/emperor/pull/405).**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false,
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "outputs": [],
   "source": [
    "import pandas as pd, numpy as np\n",
    "from emperor import Emperor, nbinstall\n",
    "from skbio import OrdinationResults\n",
    "\n",
    "from emperor.qiime_backports.parse import parse_mapping_file\n",
    "from emperor.qiime_backports.format import format_mapping_file\n",
    "\n",
    "from skbio.io.util import open_file\n",
    "\n",
    "nbinstall()\n",
    "\n",
    "def load_mf(fn):\n",
    "    with open_file(fn) as f:\n",
    "        mapping_data, header, _ = parse_mapping_file(f)\n",
    "        _mapping_file = pd.DataFrame(mapping_data, columns=header)\n",
    "        _mapping_file.set_index('SampleID', inplace=True)\n",
    "    return _mapping_file\n",
    "\n",
    "def write_mf(f, _df):\n",
    "    with open(f, 'w') as fp:\n",
    "        lines = format_mapping_file(['SampleID'] + _df.columns.tolist(),\n",
    "                                    list(_df.itertuples()))\n",
    "        fp.write(lines+'\\n')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We are going to load data from [Fierer et al. 2010](http://www.pnas.org/content/107/14/6477.full) (the data was retrieved from study [232](https://qiita.ucsd.edu/study/description/232) in [Qiita](https://qiita.ucsd.edu), remember you need to be logged in to access the study).\n",
    "\n",
    "Specifically, here we will reproduce *Figure 1 A*."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false,
    "slideshow": {
     "slide_type": "slide"
    }
   },
   "outputs": [],
   "source": [
    "mf = load_mf('keyboard/mapping-file.txt')\n",
    "res = OrdinationResults.read('keyboard/unweighted-unifrac.even1000.txt')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "# change the remote parameter to False/True depending on what you want to do\n",
    "x = Emperor(res, mf, remote=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "<script type=\"text/javascript\">\n",
       "\n",
       "if ($(\"#emperor-css\").length == 0){{\n",
       "    $(\"head\").append([\n",
       "\n",
       "        '<link id=\"emperor-css\" rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/css/emperor.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/jquery-ui.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/slick.grid.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/spectrum.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/chosen.min.css\">',\n",
       "        '<link rel=\"stylesheet\" type=\"text/css\" href=\"https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/css/jquery.contextMenu.min.css\">'\n",
       "    ]);\n",
       "}}\n",
       "</script>\n",
       "\n",
       "<div id='emperor-notebook-0x422a5d93' style=\"position: relative; width:100%; height:500px;\">\n",
       "  <div class='loading' style=\"position: absolute;top: 50%;left: 50%;margin-left: -229px; margin-top: -59px; z-index: 10000;height:118px;width:458px;padding:0px\"><img src='https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/img/emperor.png' alt='Emperor resources missing. Expected them to be found in https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files'></div>\n",
       "</div>\n",
       "</div>\n",
       "\n",
       "<script type=\"text/javascript\">\n",
       "requirejs.config({\n",
       "// the left side is the module name, and the right side is the path\n",
       "// relative to the baseUrl attribute, do NOT include the .js extension\n",
       "'paths': {\n",
       "  /* jQuery */\n",
       "  'jquery': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery-2.1.4.min',\n",
       "  'jqueryui': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery-ui.min',\n",
       "  'jquery_drag': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery.event.drag-2.2.min',\n",
       "\n",
       "  /* jQuery plugins */\n",
       "  'chosen': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/chosen.jquery.min',\n",
       "  'spectrum': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/spectrum.min',\n",
       "  'position': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery.ui.position.min',\n",
       "  'contextmenu': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/jquery.contextMenu.min',\n",
       "\n",
       "  /* other libraries */\n",
       "  'underscore': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/underscore-min',\n",
       "  'chroma': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/chroma.min',\n",
       "  'filesaver': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/FileSaver.min',\n",
       "  'blob': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/Blob',\n",
       "  'canvastoblob': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/canvas-toBlob',\n",
       "  'd3': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/d3.min',\n",
       "\n",
       "  /* THREE.js and plugins */\n",
       "  'three': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.min',\n",
       "  'orbitcontrols': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.js-plugins/OrbitControls',\n",
       "  'projector': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.js-plugins/Projector',\n",
       "  'svgrenderer': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.js-plugins/SVGRenderer',\n",
       "  'canvasrenderer': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/three.js-plugins/CanvasRenderer',\n",
       "\n",
       "  /* SlickGrid */\n",
       "  'slickcore': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/slick.core.min',\n",
       "  'slickgrid': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/slick.grid.min',\n",
       "  'slickformatters': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/slick.editors.min',\n",
       "  'slickeditors': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/vendor/js/slick.formatters.min',\n",
       "\n",
       "  /* Emperor's objects */\n",
       "  'util': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/util',\n",
       "  'model': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/model',\n",
       "  'view': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/view',\n",
       "  'controller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/controller',\n",
       "  'draw': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/draw',\n",
       "  'scene3d': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/sceneplotview3d',\n",
       "  'abcviewcontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/abc-view-controller',\n",
       "  'viewcontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/view-controller',\n",
       "  'colorviewcontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/color-view-controller',\n",
       "  'visibilitycontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/visibility-controller',\n",
       "  'scaleviewcontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/scale-view-controller',\n",
       "  'shapecontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/shape-controller',\n",
       "  'axescontroller': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/axes-controller',\n",
       "  'shape-editor': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/shape-editor',\n",
       "  'color-editor': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/color-editor',\n",
       "  'scale-editor': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/scale-editor',\n",
       "  'shapes': 'https://cdn.rawgit.com/biocore/emperor/new-api/emperor/support_files/js/shapes'\n",
       "},\n",
       "/*\n",
       "   Libraries that are not AMD compatible need shim to declare their\n",
       "   dependencies.\n",
       " */\n",
       "'shim': {\n",
       "  'jquery_drag': {\n",
       "    'deps': ['jquery', 'jqueryui']\n",
       "  },\n",
       "  'chosen': {\n",
       "    'deps': ['jquery'],\n",
       "    'exports': 'jQuery.fn.chosen'\n",
       "  },\n",
       "  'contextmenu' : {\n",
       "    'deps': ['jquery', 'jqueryui', 'position']\n",
       "  },\n",
       "  'filesaver' : {\n",
       "    'deps': ['blob']\n",
       "  },\n",
       "  'canvastoblob' : {\n",
       "    'deps': ['blob']\n",
       "  },\n",
       "  'orbitcontrols': {\n",
       "    'deps': ['three']\n",
       "  },\n",
       "  'projector': {\n",
       "    'deps': ['three']\n",
       "  },\n",
       "  'svgrenderer': {\n",
       "    'deps': ['orbitcontrols', 'projector']\n",
       "  },\n",
       "  'canvasrenderer': {\n",
       "    'deps': ['orbitcontrols', 'projector']\n",
       "  },\n",
       "'slickcore': ['jqueryui'],\n",
       "'slickgrid': ['slickcore', 'jquery_drag', 'slickformatters',\n",
       "              'slickeditors']\n",
       "}\n",
       "});\n",
       "\n",
       "requirejs(\n",
       "[\"jquery\", \"model\", \"controller\"],\n",
       "function($, model, EmperorController) {\n",
       "  var DecompositionModel = model.DecompositionModel;\n",
       "\n",
       "  var div = $('#emperor-notebook-0x422a5d93');\n",
       "\n",
       "  var data = {\n",
       "    // coordinates information\n",
       "    'sample_ids': [\"232.M3Rkey217\", \"232.M9Thmr217\", \"232.M9Pinr217\", \"232.M3Rinr217\", \"232.M2Enter217\", \"232.M9Ekey217\", \"232.M3Midr217\", \"232.M3Tkey217\", \"232.M2Rsft217\", \"232.M2Okey217\", \"232.M3Ckey217\", \"232.M9Akey217\", \"232.M9Hkey217\", \"232.M9Pinl217\", \"232.M2Kkey217\", \"232.M9Midl217\", \"232.M9Rinl217\", \"232.M9Ckey217\", \"232.M2Midr217\", \"232.M2Lsft217\", \"232.M9Mkey217\", \"232.M9Enter217\", \"232.M9Bkey217\", \"232.M9Fkey217\", \"232.M2Mkey217\", \"232.M3Wkey217\", \"232.M9Gkey217\", \"232.M3Midl217\", \"232.M3Pinl217\", \"232.M9Vkey217\", \"232.M9Wkey217\", \"232.M2Tkey217\", \"232.M3Indl217\", \"232.M9Dkey217\", \"232.M2Indl217\", \"232.M2Hkey217\", \"232.M2Gkey217\", \"232.M2Jkey217\", \"232.M9Nkey217\", \"232.M2Ukey217\", \"232.M2Fkey217\", \"232.M3Lkey217\", \"232.M9Pkey217\", \"232.M2Ykey217\", \"232.M2Pinr217\", \"232.M2Ikey217\", \"232.M2Ekey217\", \"232.M9Midr217\", \"232.M3Space217\", \"232.M9Thml217\", \"232.M9Rinr217\", \"232.M3Mkey217\", \"232.M3Rsft217\", \"232.M2Lkey217\", \"232.M2Space217\", \"232.M2Vkey217\", \"232.M3Gkey217\", \"232.M9Indr217\", \"232.M3Xkey217\", \"232.M3Ykey217\", \"232.M3Vkey217\", \"232.M2Bkey217\", \"232.M3Pinr217\", \"232.M3Bkey217\", \"232.M2Akey217\", \"232.M3Indr217\", \"232.M3Thmr217\", \"232.M2Skey217\", \"232.M2Wkey217\", \"232.M2Rinr217\", \"232.M2Ckey217\", \"232.M3Ekey217\", \"232.M2Zkey217\", \"232.M2Thml217\", \"232.M2Rinl217\", \"232.M3Rinl217\", \"232.M2Midl217\"],\n",
       "    'coordinates': [[-0.4074406818457083, -0.43896550974221316, -0.25468844457202927, 0.06719207132574237, 0.16393776044523461], [-0.4207394101358396, 0.4127619282873838, -0.43855428124270796, -0.13362165721928865, -0.029649859291289184], [-0.40108307080085, 0.45807964103487414, -0.28907254962003476, 0.13884579342831826, -0.18404409935138255], [-0.22607103975893267, -0.5437679949836518, -0.23460673217971084, 0.07042446208054179, -0.30030978960720833], [0.6976164237026521, -0.03249256188839796, -0.07467366361482833, -0.21917176331415086, -0.245579902827961], [-0.5040232369250873, 0.1785136879573188, 0.42267760006015553, -0.39270232197680455, 0.006390953901908871], [-0.3165699961956398, -0.5243822057231243, 0.017659744520787307, 0.08343531857872967, -0.3144271178392954], [-0.47864428895259736, -0.17888159629040717, -0.021415907070522686, 0.08616645406719844, 0.21501461162754318], [0.7486793320731893, 0.09440778602758665, 0.04923273466953846, 0.11688343982972595, 0.04885405891702237], [0.46373134324548665, -0.11389940568913652, -0.09017594389952956, -0.34985487344761174, 0.020165815797683288], [-0.5309536314218456, -0.2764989478374162, 0.562701647323495, 0.07066084840943682, 0.21788556256368338], [-0.41878610188903004, 0.2440323617326647, 0.6454795456795647, -0.2072723297714295, -0.12166265852045095], [-0.5696380137810988, 0.4912435955197167, 0.24242282855184988, -0.07152391130592768, -0.129874294341997], [-0.40190699787794104, 0.35157696713683606, -0.45733181925630245, 0.005758608744212644, -0.10912951596839811], [0.6333926006501015, 0.10125426049770483, -0.05804920291813072, -0.1929372353605661, -0.14666650259940128], [-0.49554583367357724, 0.34328869301885545, -0.25586141939046403, 0.003832146191899642, 0.04216167676740839], [-0.40204108444094133, 0.11498883194063915, -0.41170050661157814, -0.15827235905166215, 0.07032118699845355], [-0.609534516623967, 0.3223784222841321, 0.1353336606115565, -0.05070038860879401, 0.04842354646998965], [0.45428965098143237, 0.043314004809713334, -0.2935562424507357, 0.04978458024111463, -0.042433756652518625], [0.6491851015420564, 0.03742136739982878, -0.06574688557723682, -0.16407261066414952, 0.05203279261327233], [-0.5912899447840353, 0.2805666047406449, -0.04315640007383779, 0.0366620643266138, 0.05268723435223584], [-0.6004107366952997, 0.3136748602295868, 0.3210792738808785, -0.10965811423801282, -0.0671235003545022], [-0.6272917710675684, 0.35978327198328597, 0.10039288198770172, -0.042675117980398775, 0.10504378382571224], [-0.5971224169828981, 0.3123438858946691, 0.12299674051392472, -0.18281238426948554, 0.3066571740442304], [0.3098458577241026, 0.022734417519609147, -0.13587385872129795, -0.16090085236253976, 0.23192275943868182], [-0.42581469112859244, -0.3400579449851507, -0.12479701698014932, 0.22999395932116212, 0.45208306218251204], [-0.517645834808039, 0.3182703281396283, 0.37438669874695824, -0.16348251962681992, -0.19887336133077144], [-0.3766195614077237, -0.1367835409159127, -0.21381459676059036, 0.13491753082158825, -0.1984114833620629], [-0.09549020001565454, -0.5558316218357581, -0.1706807560881345, 0.17555993028252503, -0.17901901215816926], [-0.48906465036191205, 0.3100848778993016, 0.13984597171924223, -0.08987140362246421, -0.0422840202062092], [-0.5927781148343384, 0.4210775999612222, 0.12883627351889151, 0.02071306518255713, -0.010911743317364159], [0.7539680390697676, 0.014857961127467319, 0.06067021546732213, -0.05812789214178965, 0.15809165052479157], [-0.4755272270775535, -0.17048384933304467, -0.26536847117384116, 0.14363160183302948, 0.21197104282512588], [-0.5719995975962872, 0.35338988815994266, 0.2112259293087332, -0.20553378402228442, 0.04273730436920184], [0.5301204747925141, 0.05391912124171768, -0.06585987659310755, 0.04000846386672511, 0.0935418653686436], [0.5711693474509173, -0.05496725201723638, -0.1835501157767039, -0.39800373158825714, 0.24234526742304363], [0.7828931762395881, 0.027713054788288814, -0.2727763252643691, -0.08669428851897894, 0.06941548153870328], [0.5771857134818571, -0.17791340477608325, -0.10680785503673473, -0.5409639534789471, 0.2512084439463247], [-0.6572515785069658, 0.2810175311567659, 0.44154406206702157, -0.1761726923026677, -0.15222810828264374], [0.43956803541614314, -0.04457363525217865, -0.14373744887276851, -0.20726116207113435, -0.019955530626788617], [0.9024799577495102, -0.009192403590453925, -0.08904329918166892, -0.14666856272051265, -0.09716796801973648], [-0.32497398823883156, -0.5705144552442156, 0.057779052268499005, -0.05814631090806475, 0.14439244143101507], [-0.6058290492790207, 0.08489121715938414, 0.7174012874196465, -0.18490195317835723, -0.12918021974625923], [0.6264554962289707, -0.207723371779376, -0.017795048816653888, -0.35535333139516234, 0.1864116837725768], [0.46306239032512303, 0.03960064764667467, -0.02263177109943106, -0.15232187530529426, 0.11135813348787099], [0.6316553386893512, -0.1025901796987955, 0.22655547475339094, -0.017533855329317005, -0.28259468551407607], [0.9468837150038528, 0.12305950672810953, 0.24068331298654966, 0.3507222876248546, -0.23164576015325972], [-0.4984928244009464, 0.4832262304562883, -0.548750838296117, 0.09835310839103223, -0.0861279632234858], [-0.4928272860161017, -0.5695310569132762, 0.18366969183057277, -0.05064160551867323, -0.08065529326772433], [-0.6152543908355868, 0.6438765174643952, -0.19181771598093064, 0.08756785981435149, -0.06270168126034611], [-0.41930673553140724, 0.5428675647647156, -0.5816880564645296, 0.12075651775218767, -0.0981347584627667], [-0.3843292124763502, -0.5448570419514451, 0.3552837555114072, -0.12228185868477885, 0.1558745291834206], [-0.3326005936728958, -0.33239646597873646, -0.027102678696207347, 0.12052201255236954, -0.35599877571403776], [0.4765852111884073, -0.11297052374389564, 0.0035884135027534894, -0.22401710952749126, 0.2841432313296042], [0.9055817352164423, -0.06729295464963356, -0.00917814720895573, -0.24990868920015707, -0.22644293776252974], [0.9403149956684648, 0.061663046054773275, -0.025620299558803763, -0.06273442300648355, -0.02557221029051302], [-0.33590115196974474, -0.31508878896018677, -0.16513918979518044, 0.04308592731877219, 0.43309838777846843], [-0.512324675503843, 0.5378708814477152, -0.43653181539177394, 0.07450506478998953, -0.11888761846897748], [-0.32129315098710487, -0.5704354972577514, -0.1735846823645882, -0.012743758285989218, -0.15065644013914298], [-0.41210135842481366, -0.42254075958752624, 0.011542684976365463, -0.10452360977272862, -0.18877314419952748], [-0.5140524512036688, -0.2878958176468228, 0.14433773868105093, 0.14580634051617214, 0.32251528856333883], [0.8113930628036884, 0.15893117912228003, -0.10241880040036368, -0.05008415993298737, 0.06561504746716873], [-0.3515448253612688, -0.5961276469287253, -0.062504494214491, 0.10640792949302248, -0.44778138479902047], [-0.4664199203581045, -0.27251893100675234, -0.03161307100707661, 0.21222365058463627, 0.357964278052898], [0.8528759092494218, 0.08097071301016512, 0.009593892936712912, 0.06339970762251079, -7.825809187388054e-05], [-0.2730843188810611, -0.646132331196342, -0.03829914009767361, 0.11171223325237903, -0.09917711659190541], [-0.3632682931924479, -0.45157332031286096, -0.18509367098284366, 0.2506258178033899, -0.286341326105752], [0.6881613923314701, 0.058423547863209316, 0.20846204581821984, 0.3505177144220234, 0.11347203913883584], [0.8849297447952548, 0.04728077982224395, 0.17128617675073052, -0.05666138206604238, -0.097971510778778], [0.4895982260581956, 0.05539752276650165, -0.12193164194549438, -0.025600422361037146, 0.1667812198372837], [1.0, 0.20775732203592487, 0.26043321348398124, 0.2839605604764679, -0.20517935150316213], [-0.5138240578760805, -0.3282705123977203, 0.11624867386653111, 0.1658638254180597, 0.059780201739507925], [0.7846377901684469, -0.0352736184684172, 0.12176467586271984, -0.1134511889228628, -0.1862938395970687], [0.7710098590859557, 0.2308188543200622, 0.23438712411179632, 0.48662470643604194, 0.005772866956493504], [0.5707233753505154, 0.31403235814983843, 0.2833702504440817, 0.7088322119301717, 0.2539683031316246], [-0.43669520998556777, -0.21067661868170928, -0.09806946559892735, 0.2752888498931831, 0.030835464518508184], [0.6174144275018919, 0.309738927962683, 0.2777968730144237, 0.8186147784373727, 0.15507034799833358]],\n",
       "    'axes_names': [0, 1, 2, 3, 4],\n",
       "    'percents_explained': [23.330988966275164, 7.445584042747666, 4.615896035685128, 3.4824013326861176, 2.5304874508573096],\n",
       "\n",
       "    // sample information\n",
       "    'metadata_headers': [\"SampleID\", \"BarcodeSequence\", \"LinkerPrimerSequence\", \"center_name\", \"center_project_name\", \"emp_status\", \"experiment_design_description\", \"key_seq\", \"library_construction_protocol\", \"linker\", \"platform\", \"region\", \"run_center\", \"run_date\", \"run_prefix\", \"samp_size\", \"sample_center\", \"sequencing_meth\", \"study_center\", \"target_gene\", \"target_subfragment\", \"age\", \"age_unit\", \"altitude\", \"anonymized_name\", \"assigned_from_geo\", \"body_habitat\", \"body_product\", \"body_site\", \"collection_timestamp\", \"country\", \"depth\", \"dna_extracted\", \"elevation\", \"env_biome\", \"env_feature\", \"env_matter\", \"has_physical_specimen\", \"host_subject_id\", \"host_taxid\", \"latitude\", \"longitude\", \"physical_specimen_remaining\", \"project_name\", \"required_sample_info_status\", \"sample_type\", \"sex\", \"taxon_id\", \"title\", \"Description\"],\n",
       "    'metadata': [[\"232.M3Rkey217\", \"AGTCCATAGCTG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Rkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Rkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Rkey\"], [\"232.M9Thmr217\", \"ACTACGTGTGGT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Thmr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M9Pinr217\", \"ACTCAGATACTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Pinr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Rinr217\", \"AGGTGTGATCGC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Rinr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Enter217\", \"ACGGTGAGTGTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Enter217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ente\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ente\"], [\"232.M9Ekey217\", \"ACTCTTCTAGAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Ekey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ekey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ekey\"], [\"232.M3Midr217\", \"AGGCTACACGAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Midr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Tkey217\", \"AGTCTACTCTGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Tkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Tkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Tkey\"], [\"232.M2Rsft217\", \"ACGCTCATGGAT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Rsft217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Right_shift\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Right_shift\"], [\"232.M2Okey217\", \"ACAGTTGCGCGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Okey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Okey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Okey\"], [\"232.M3Ckey217\", \"ATAGGCGATCTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Ckey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ckey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ckey\"], [\"232.M9Akey217\", \"AGACCGTCAGAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Akey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Akey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Akey\"], [\"232.M9Hkey217\", \"AGAGTAGCTAAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Hkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Hkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Hkey\"], [\"232.M9Pinl217\", \"ACTACAGCCTAT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Pinl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Kkey217\", \"ACCTCGATCAGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Kkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Kkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Kkey\"], [\"232.M9Midl217\", \"ACGTGCCGTAGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Midl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M9Rinl217\", \"ACGTTAGCACAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Rinl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M9Ckey217\", \"AGCACACCTACA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Ckey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ckey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ckey\"], [\"232.M2Midr217\", \"ACACACTATGGC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Midr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Lsft217\", \"ACGGATCGTCAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Lsft217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Left_shift\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Left_shift\"], [\"232.M9Mkey217\", \"AGCATATGAGAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Mkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Mkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Mkey\"], [\"232.M9Enter217\", \"AGCGAGCTATCT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Enter217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ente\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ente\"], [\"232.M9Bkey217\", \"AGCAGCACTTGT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Bkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Bkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Bkey\"], [\"232.M9Fkey217\", \"AGAGAGCAAGTG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Fkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Fkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Fkey\"], [\"232.M2Mkey217\", \"ACGCTATCTGGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Mkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Mkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Mkey\"], [\"232.M3Wkey217\", \"AGTAGTATCCTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Wkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Wkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Wkey\"], [\"232.M9Gkey217\", \"AGAGCAAGAGCA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Gkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Gkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Gkey\"], [\"232.M3Midl217\", \"AGCTCCATACAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Midl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Pinl217\", \"AGCTGACTAGTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Pinl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M9Vkey217\", \"AGCACGAGCCTA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Vkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Vkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Vkey\"], [\"232.M9Wkey217\", \"ACTCGCACAGGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Wkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Wkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Wkey\"], [\"232.M2Tkey217\", \"ACAGAGTCGGCT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Tkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Tkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Tkey\"], [\"232.M3Indl217\", \"AGCTATCCACGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Indl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M9Dkey217\", \"AGACTGCGTACT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Dkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Dkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Dkey\"], [\"232.M2Indl217\", \"AACTCGTCGATG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Indl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Hkey217\", \"ACCAGCGACTAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Hkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Hkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Hkey\"], [\"232.M2Gkey217\", \"ACCAGACGATGC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Gkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Gkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Gkey\"], [\"232.M2Jkey217\", \"ACCGCAGAGTCA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Jkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Jkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Jkey\"], [\"232.M9Nkey217\", \"AGCAGTCGCGAT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Nkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Nkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Nkey\"], [\"232.M2Ukey217\", \"ACAGCTAGCTTG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Ukey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ukey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ukey\"], [\"232.M2Fkey217\", \"ACCACATACATC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Fkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Fkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Fkey\"], [\"232.M3Lkey217\", \"ATACTATTGCGC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Lkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Lkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Lkey\"], [\"232.M9Pkey217\", \"AGAACACGTCTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M9Pkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M9\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Pkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Pkey\"], [\"232.M2Ykey217\", \"ACAGCAGTGGTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Ykey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ykey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ykey\"], [\"232.M2Pinr217\", \"ACACGAGCCACA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Pinr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Ikey217\", \"ACAGTGCTTCAT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Ikey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ikey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ikey\"], [\"232.M2Ekey217\", \"ACACTGTTCATG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Ekey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ekey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ekey\"], [\"232.M9Midr217\", \"ACTATTGTCACG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Midr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Space217\", \"ATCGATCTGTGG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Space217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Space_bar\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Space_bar\"], [\"232.M9Thml217\", \"ACGTCTGTAGCA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Thml217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M9Rinr217\", \"ACTCACGGTATG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Rinr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Mkey217\", \"ATCACTAGTCAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Mkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Mkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Mkey\"], [\"232.M3Rsft217\", \"ATCCGATCACAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Rsft217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Right_shift\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Right_shift\"], [\"232.M2Lkey217\", \"ACCTGTCTCTCT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Lkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Lkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Lkey\"], [\"232.M2Space217\", \"ACGTACTCAGTG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Space217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Space_bar\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Space_bar\"], [\"232.M2Vkey217\", \"ACGCAACTGCTA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Vkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Vkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Vkey\"], [\"232.M3Gkey217\", \"ATAATCTCGTCG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Gkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Gkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Gkey\"], [\"232.M9Indr217\", \"ACTAGCTCCATA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"25\", \"years\", \"0\", \"M9Indr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M9\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Xkey217\", \"ATAGCTCCATAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Xkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Xkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Xkey\"], [\"232.M3Ykey217\", \"AGTCTCGCATAT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Ykey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ykey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ykey\"], [\"232.M3Vkey217\", \"ATATCGCTACTG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Vkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Vkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Vkey\"], [\"232.M2Bkey217\", \"ACGCGATACTGG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Bkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Bkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Bkey\"], [\"232.M3Pinr217\", \"AGTACGCTCGAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Pinr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Bkey217\", \"ATATGCCAGTGC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Bkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Bkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Bkey\"], [\"232.M2Akey217\", \"ACATGATCGTTC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Akey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Akey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Akey\"], [\"232.M3Indr217\", \"AGGACGCACTGT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Indr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Thmr217\", \"AGCTTGACAGCT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Thmr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Skey217\", \"ACATGTCACGTG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Skey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Skey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Skey\"], [\"232.M2Wkey217\", \"ACACTAGATCCG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Wkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Wkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Wkey\"], [\"232.M2Rinr217\", \"ACACATGTCTAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Rinr217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Ckey217\", \"ACGATGCGACCA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Ckey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ckey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ckey\"], [\"232.M3Ekey217\", \"AGTCACATCACT\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M3Ekey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M3\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Ekey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Ekey\"], [\"232.M2Zkey217\", \"ACGACGTCTTAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"unknown\", \"years\", \"0\", \"M2Zkey217\", \"False\", \"unknown\", \"unknown\", \"unknown\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:surface\", \"ENVO:surface\", \"ENVO:surface\", \"True\", \"M2\", \"36244\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"Zkey\", \"male\", \"408169\", \"Forensic_identification_using_skin_bacterial_communities\", \"Zkey\"], [\"232.M2Thml217\", \"AACGCACGCTAG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Thml217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Rinl217\", \"AAGAGATGTCGA\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Rinl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M3Rinl217\", \"AGCTCTCAGAGG\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"33\", \"years\", \"0\", \"M3Rinl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M3\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"], [\"232.M2Midl217\", \"AACTGTGCGTAC\", \"CATGCTGCCTCCCGTAGGAGT\", \"CCME\", \"Forensic_identification_using_skin_bacterial_communities\", \"EMP\", \"Forensic_identification_using_skin_bacterial_communities\", \"TCAG\", \"16S_rRNA_gene_sequences_were_processed_according_to_the_methods_described_in_our_previous_publications_(Fierer_et_al.,_2008;_Hamady_et_al.,_2008)._Briefly,_sequences_\\u003c200_or_\\u003e300gnt_or_with_average_quality_scores_of_\\u003c25_were_removed_from_the_dataset,_as_were_those_with_uncorrectable_barcodes,_ambiguous_bases,_or_if_the_bacterial_16S_rRNA_gene-specific_primer_was_absent._Sequences_were_then_assigned_to_the_specific_subsamples_based_on_their_unique_12nt_barcode_and_then_grouped_into_phylotypes_at_the_97%_level_of_sequence_identity_using_cd-hit_(Li_\\u0026_Godzik,_2006)_with_a_minimum_coverage_of_97%._We_chose_to_group_the_phylotypes_at_97%_identity_because_this_matches_the_limits_of_resolution_of_pyrosequencing_(Kunin_et_al.,_2010)_and_because_the_branch_length_so_omitted_contributes_little_to_the_tree_and_therefore_to_phylogenetic_estimates_of___diversity_(Hamady_et_al.,_2009)._A_representative_for_each_phylotype_was_chosen_by_selecting_the_most_abundant_sequence_in_the_phylotype,_with_ties_being_broken_by_choosing_the_longest_sequence._A_phylogenetic_tree_of_the_representative_sequences_was_constructed_using_the_Kimura_2-parameter_model_in_Fast_Tree_(Price_et_al.,_2009)_after_sequences_were_aligned_with_NAST_(minimum_150nt_at_75%_minimum_identity)_(DeSantis_et_al.,_2006a)_against_the_GreenGenes_database_(DeSantis_et_al.,_2006b)._Hypervariable_regions_were_screened_out_of_the_alignment_using_PH_Lane_mask_(http://greengenes.lbl.gov/)._Differences_in_the_community_composition_for_each_pair_of_samples_were_determined_from_the_phylogenetic_tree_using_the_weighted_and_unweighted_UniFrac_algorithms_(Lozupone_\\u0026_Knight,_2005;_Lozupone_et_al.,_2006)._UniFrac_is_a_tree-based_metric_that_measures_the_distance_between_two_communities_as_the_fraction_of_branch_length_in_a_phylogenetic_tree_that_is_unique_to_one_of_the_communities_(as_opposed_to_being_shared_by_both)._This_method_of_community_comparison_accounts_for_the_relative_similarities_and_differences_among_phylotypes_(or_higher_taxa)_rather_than_treating_all_taxa_at_a_given_level_of_divergence_as_equal_(Lozupone_\\u0026_Knight,_2008)._Although_UniFrac_depends_on_a_phylogenetic_tree,_it_is_relatively_robust_to_differences_in_the_tree_reconstruction_method_or_to_the_approximation_of_using_phylotypes_to_represent_groups_of_very_similar_sequences_(Hamady_et_al.,_2009).\", \"CA\", \"FLX\", \"0\", \"CCME\", \"8/14/08\", \"FFCKVMW\", \"1, swab\", \"CCME\", \"pyrosequencing\", \"CCME\", \"16S rRNA\", \"V2\", \"36\", \"years\", \"0\", \"M2Midl217\", \"False\", \"UBERON:skin\", \"UBERON:sebum\", \"UBERON:skin\", \"7/15/08\", \"GAZ:United States of America\", \"0\", \"True\", \"1624\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"ENVO:human-associated habitat\", \"True\", \"M2\", \"9606\", \"40.0083\", \"-105.2705\", \"False\", \"fierer_forensic_keyboard\", \"completed\", \"finger_tip\", \"male\", \"539655\", \"Forensic_identification_using_skin_bacterial_communities\", \"finger_tip\"]]\n",
       "  }\n",
       "\n",
       "  var dm, ec;\n",
       "\n",
       "  function init() {\n",
       "    // Initialize the DecompositionModel\n",
       "    dm = new DecompositionModel('', data['sample_ids'],\n",
       "                                data['coordinates'],\n",
       "                                data['percents_explained'],\n",
       "                                data['metadata_headers'], data['metadata'],\n",
       "                                data['axes_names']);\n",
       "    // Initialize the EmperorController\n",
       "    ec = new EmperorController(dm, \"emperor-notebook-0x422a5d93\");\n",
       "  }\n",
       "\n",
       "  function animate() {\n",
       "    requestAnimationFrame(animate);\n",
       "    ec.render();\n",
       "  }\n",
       "  $(window).resize(function() {\n",
       "    ec.resize(div.innerWidth(), div.innerHeight());\n",
       "  });\n",
       "\n",
       "  $(function(){\n",
       "    init();\n",
       "    animate();\n",
       "\n",
       "  });\n",
       "\n",
       "}); // END REQUIRE.JS block\n",
       "</script>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": [
       "<emperor.core.Emperor at 0x106b40128>"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "x"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.1"
  },
  "widgets": {
   "state": {},
   "version": "1.1.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
